cli/src/commands/workspace.rs

```diff
@@ -61,10 +61,22 @@ pub(crate) struct WorkspaceAddArgs {
     /// directory.
     #[arg(long)]
     name: Option<String>,
-    /// The revision that the workspace should be created at; a new working copy
-    /// commit will be created on top of it.
+    /// A list of parent revisions for the new workspace and its working-copy
+    /// commit. May be empty, a single revision, or multiple revisions.
+    ///
+    /// If no revision is provided (the default), the new workspace will be
```

Comment:
nit: "(the default)" here confused me a bit because it feels like "no revision is provided" is the definition of "default", so it made me stop and wonder how if you can explicitly pass no revisions somehow. How about "By default (i.e. if no revisions were specified), ..."? I understand that it doesn't match the paragraphs below as well then. Another option is to just drop "(the default)".